pieman.sh

```diff
@@ -157,13 +157,13 @@ run_scripts "bootstrap"
 umount_required_filesystems
 
 info "creating image"
-dd if=/dev/zero of=${IMAGE} bs=1024 seek=$[ 1024 * 1024 * 7 ] count=1
+dd if=/dev/zero of=${IMAGE} bs=1024 seek=$[ `du -s ${R}/ | cut -f1` + 150 * 1024 ] count=1
 
 parted ${IMAGE} mktable msdos
 
-parted ${IMAGE} mkpart p fat32 4MiB 54MiB
+parted ${IMAGE} mkpart p fat32 4MiB 104MiB
 
-parted -s ${IMAGE} -- mkpart primary ext2 58MiB -1s
+parted -s ${IMAGE} -- mkpart primary ext2 104MiB -1s
```

Comment:
Let's align the second partition on 4MiB boundary too: 
`parted -s ${IMAGE} -- mkpart primary ext2 108MiB -1s`

Reply:
104MB is aligned tho, do we have to leave a gap?